apps/challenges/serializers.py

```diff
@@ -63,11 +64,55 @@ class ChallengePhaseSplitSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = ChallengePhaseSplit
-        fields = '__all__'
         fields = ('id', 'dataset_split', 'challenge_phase', 'challenge_phase_name', 'dataset_split_name', 'visibility')
 
     def get_dataset_split_name(self, obj):
         return obj.dataset_split.name
 
     def get_challenge_phase_name(self, obj):
         return obj.challenge_phase.name
+
+
+class ChallengeConfigurationSerializer(serializers.ModelSerializer):
+    """
+    Serialize the ChallengeConfiguration Model.
+    """
+    def __init__(self, *args, **kwargs):
+        super(ChallengeConfigurationSerializer, self).__init__(*args, **kwargs)
+        context = kwargs.get('context')
+        if context:
+            user = context.get('user')
+            kwargs['data']['user'] = user.pk
+
+    class Meta:
+        model = ChallengeConfiguration
+        fields = ('zip_configuration', 'user',)
+
+
+class LeaderboardSerializer(serializers.ModelSerializer):
```

Comment:
Actually , this PR is adding all the serializers that are needed during the creation of challenge using zip file and as we are creating the Leaderboard model instance there , so I added the serializer for it. Should i create a separate PR for this?